### PR TITLE
bundle: Add missing CRDs to bundle CSV

### PR DESCRIPTION
**Pull request by black-dragon74**
This PR adds the EncryptionKeyRotation and
VolumeGroupReplication CRDs to the bundle's CSV.

### CODE REVIEW

File: config/manifests/bases/clusterserviceversion.yaml.in [hunk 1]
```diff
@@ -31,6 +31,16 @@ spec:
       kind: ReclaimSpaceJob
       name: reclaimspacejobs.csiaddons.openshift.io
       version: v1alpha1
+    - description: EncryptionKeyRotationCronJob is the Schema for the encryptionkeyrotationcronjobs API
```
Comment (Madhu-1) [inline config/manifests/bases/clusterserviceversion.yaml.in:34]: @black-dragon74 can you please open an issue for this one, In our CI we should identify this kind of inconsistency

**Review by Madhu-1 — COMMENTED** [review]
Please update the PR description and commit about the command you ran to generate it

**Comment by black-dragon74** [comment]
> Please update the PR description and commit about the command you ran to generate it

The PR modifies the template used for bundle CSV and does not include any generated files.

File: config/manifests/bases/clusterserviceversion.yaml.in (hunk 1 above)
Comment (black-dragon74) [reply config/manifests/bases/clusterserviceversion.yaml.in:34]: Done: https://github.com/csi-addons/kubernetes-csi-addons/issues/670

Comment (Madhu-1) [reply config/manifests/bases/clusterserviceversion.yaml.in:34]: can you please add the volumegroupreplication CRDs as well, it looks like that is also missing.

Comment (black-dragon74) [reply config/manifests/bases/clusterserviceversion.yaml.in:34]: Updated commit desc as well :)

**Review by Madhu-1 — APPROVED** [review]

**Review by Rakshith-R — APPROVED** [review]

**Pull request merged**